### PR TITLE
feat: add parameters to the TierTemplateRevision CRD

### PR DESCRIPTION
**Pull request by mfrancisc**
generated from https://github.com/codeready-toolchain/api/pull/451

### CODE REVIEW

File: config/crd/bases/toolchain.dev.openshift.com_tiertemplaterevisions.yaml [hunk 1]
```diff
@@ -4,7 +4,6 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.12.0
-  creationTimestamp: null
```
Comment (mfrancisc) [inline config/crd/bases/toolchain.dev.openshift.com_tiertemplaterevisions.yaml:7]: not sure why this has changed 🤔

Comment (MatousJobanek) [reply config/crd/bases/toolchain.dev.openshift.com_tiertemplaterevisions.yaml:7]: I guess that it's a result of a wrong conflicts solving in the @ranakan19's PR when she was updating the k8s version.

Comment (MatousJobanek) [reply config/crd/bases/toolchain.dev.openshift.com_tiertemplaterevisions.yaml:7]: see https://github.com/codeready-toolchain/host-operator/commit/c91f5c0cc879f2ee230877a04574bbd508c61f7b#diff-00ecc3755e3d97771f68b07c45dc5093ef0d59f0fd8a60e5df398c8f31f945d6L6

**Review by MatousJobanek — APPROVED** [review]

File: config/crd/bases/toolchain.dev.openshift.com_tiertemplaterevisions.yaml (hunk 1 above)
Comment (mfrancisc) [reply config/crd/bases/toolchain.dev.openshift.com_tiertemplaterevisions.yaml:7]: ok, I see that the other `creationTimestamps` were removed in the PR , but this one no. Thanks!

**Pull request merged**